src/librustc_metadata/encoder.rs

```diff
@@ -668,14 +668,15 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
             hir::ItemDefaultImpl(..) => {
                 let data = ImplData {
                     polarity: hir::ImplPolarity::Positive,
+                    defaultness: hir::Defaultness::Final,
```

Comment:
I’m not sure this PR is a right place to discuss `UnwindUnsafe`. If you have any ideas, you should post post them on the `UnwindUnsafe` issue.